src/screens/home/HomeScreen/components/TokenComponent.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from 'react';
+import {View, Text, StyleSheet, Image} from 'react-native';
+import {scale} from "device";
+import {Row, Col, CButton} from 'components';
+import {colors, images, textStyles} from "assets";
+import {toFormattedCurrency} from "utils/helpers/format";
+
+interface Props {
+    value: any
+}
+
+const TokenComponent = ({value}: Props) => {
+    return (
+        <CButton>
+            <Row.LR mx={16} style={styles.container}>
+                <Row>
+                    {
+                        value.symbol && (
+                            value.symbol === 'CSPR' ? <Image source={value.icon} style={styles.symbol}/> :
+                                <Image source={{uri: value.icon}} style={styles.symbol}/>
```

Comment:
could be sorter ?
` <Image source={value.symbol === 'CSPR' ? value.icon : {uri: value.icon}} style={styles.symbol}/>`